python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/port_channel_interfaces.schema.yml

```diff
@@ -1452,6 +1453,26 @@ keys:
                   valid_values:
                     - microseconds
                     - milliseconds
+            min_delay_dynamic:
+              description: Mutually exclusive with min_delay_static, if both are defined min_delay_static takes precedence.
+              type: dict
+              keys:
+                twamp_light_fallback:
+                  type: dict
+                  keys:
+                    number:
+                      type: int
+                      required: true
+                      convert_types:
+                        - str
+                      min: 1
+                      max: 16777215
```

Comment:
as discussed let's remove these and add some info on limit in the description for the number field.

Reply:
Implemented, thanks!